Controller/SchedulerController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Goksagun\SchedulerBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/scheduler")
+ */
+class SchedulerController extends AbstractController
+{
+    /**
+     * @Route("/run", methods={"POST"})
+     */
+    public function run(KernelInterface $kernel)
+    {
+        $application = new Application($kernel);
+        $application->setAutoExit(false);
+
+        $input = new ArrayInput([
+            'command' => 'scheduler:run'
```

Comment:
scheduler:run opsiyonlarını da inputtan query string ile alıp komuta dahil etmeliyiz örnekler: --async

Reply:
ilgili parametreler query string'den okunup command'a pass edildi. validation zaten command içinde mevcut olduğu için ek bir validation uygulanmadı.